course_catalog/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+
+class Course(models.Model):
+    title = models.CharField(max_length=256)
+    short_description = models.CharField(max_length=1024)
+    level = models.CharField(max_length=128)
+    semester = models.CharField(max_length=10)
+    language = models.CharField(max_length=128)
+    platform = models.CharField(max_length=128)
+    year = models.IntegerField()
+    id = models.CharField(max_length=128, unique=True)
```

Comment:
Assuming it's from the JSON, maybe rename it to `course_id`, django should generate it's own `id` field.

Reply:
Yes, we get an uid from both OCW and MITx. We're happy to name it `course_id` or `platform_id` and use the Django-generated auto-incrementing `id` field as the primary key instead. I don't think we have a guarantee that uids given to us will be unique across data sources, but I'm fine with having a database-enforced `unique=True` constraint for now because the schema for uids between the data sources is different (one has hypens, one does not); we can change it later if need be.